pandas-stubs/core/frame.pyi

```diff
@@ -198,7 +198,7 @@ class _LocIndexerFrame(_LocIndexer):
     def __setitem__(
         self,
         idx: MaskType | StrLike | _IndexSliceTuple | list[ScalarT],
-        value: S1 | ArrayLike | Series | DataFrame | None,
+        value: S1 | ArrayLike | Series | DataFrame | list | None,
```

Comment:
I'm not sure how strict we want to be. Could replace `list` with `Sequence[S1]`

Reply:
`Sequence[S1]` might be too narrow.  I think we would want to allow something that had mixed types.  E.g., `df.loc["a"] = [3.2, "abc"]`